app-cdr/cdrtools/cdrtools-3.01~a30.recipe

```diff
@@ -0,0 +1,160 @@
+- isodump   A program to dump ISO-9660 media 
+- isoinfo   A program to analyse/verify ISO/9660/Joliet/Rock-Ridge Filesystems 
+- isovfy    A program to verify the ISO-9660 structures 
+- rscsi     A Remote SCSI enabling daemon
+"
+HOMEPAGE="http://cdrecord.org/" 
+COPYRIGHT="
+	1995-2015 Joerg Schilling (cdrecord, readcd)
+	1993-2004 Heiko Eissfeldt, 2004-2015 Joerg Schilling (cdda2wav)
+	1993-1997 Eric Youngdale, 1997-2002 James Pearson, 1997-2015 Joerg Schilling (mkisofs)
+	"
+LICENSE="
+	GNU GPL v2
+	CDDL v1
+	"
+SOURCE_URI="http://downloads.sourceforge.net/project/cdrtools/alpha/cdrtools-3.01a30.tar.bz2"
+CHECKSUM_SHA256="5b9a2f98771c9d0097a1e7640727655ece2864eea95f38e5611af2b2f6e6d9cd"
+REVISION="1"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PATCHES='cdrtools-3.01~a30.patch'
+
+PROVIDES="
+	cdrtools = $portVersion compat >= 3
+	cmd:devdump = $portVersion compat >= 3
+	cmd:rscsi = $portVersion compat >= 3
+	cmd:scgcheck = $portVersion compat >= 3
+	cmd:mkisofs = $portVersion compat >= 3
+	cmd:btcflash = $portVersion compat >= 3
+	cmd:scgskeleton = $portVersion compat >= 3
+	cmd:isovfy = $portVersion compat >= 3
+	cmd:readcd = $portVersion compat >= 3
+	cmd:isodebug = $portVersion compat >= 3
+	cmd:cdda2mp3 = $portVersion compat >= 3
+	cmd:cdda2ogg = $portVersion compat >= 3
+	cmd:cdda2wav = $portVersion compat >= 3
+	cmd:mkhybrid = $portVersion compat >= 3
+	cmd:cdrecord = $portVersion compat >= 3
+	cmd:isodump = $portVersion compat >= 3
+	cmd:isoinfo = $portVersion compat >= 3
+	"
+REQUIRES="
+	haiku
+	"
+BUILD_REQUIRES="
+	"
+BUILD_PREREQUIRES="
+	haiku_devel
+	cmd:gcc
+	cmd:grep
+	cmd:ld
+	cmd:make
+	cmd:sed
+	"
+
+GLOBAL_WRITABLE_FILES="
+	settings/cdrecord keep-old
+	settings/rscsi keep-old
+	"
+
+SOURCE_DIR="cdrtools-3.01"
+
+patchInsdir()
+{
+	# Usage: patchInsdir <oldDir> <newDir> <file> ...
+	oldDir=$1
+	newDir=$2
+	shift 2
+
+	sed -i "s,^INSDIR=\s*$oldDir,INSDIR= $newDir," $@
+}
+
+PATCH()
+{
+	allMakefiles="$(find . -name Makefile\*) $(find . -name \*.mk)"
+
+	patchInsdir bin $relativeBinDir $allMakefiles
+	patchInsdir sbin $relativeBinDir $allMakefiles
+	patchInsdir share/doc $relativeDocDir $allMakefiles
+	patchInsdir include $relativeIncludeDir $allMakefiles
+	patchInsdir lib $relativeLibDir $allMakefiles
+	patchInsdir etc/default settings $allMakefiles
+
+	sed -i "s,/etc/default,$sysconfDir," \
+		btcflash/btcflash.1 \
+		cdda2wav/cdda2mp3 \
+		cdda2wav/cdda2ogg \
+		cdda2wav/cdda2*.1 \
+		cdrecord/COPYING \
+		cdrecord/README.* \
+		cdrecord/auinfo.c \
+		cdrecord/cdrecord.1 \
+		doc/*.man \
+		doc/*.ps \
+		include/schily/deflts.h \
+		libcdrdeflt/cdrdeflt.c \
+		libcdrdeflt/cdrdeflt.h \
+		mkisofs/diag/isoinfo.8 \
+		readcd/readcd.1 \
+		rscsi/rscsi.1 \
+		rscsi/rscsi.c \
+		scgskeleton/scgskeleton.1
+}
+
+BUILD()
+{
+	make GMAKE_NOWARN=true INS_BASE=$prefix INS_RBASE=$prefix LDPATH="" \
+		RUNPATH="" DEFMANBASE=$relativeDocumentationDir
+		# not multi-job safe
```

Comment:
I've just made the change, with some extra polishing.
I did a separate commit in order to add cross-build support. I don't know, however, if the Makefiles will support cross-building the way we would expect.
Shall I merge all the commits or should I discard the last one (which tries to add cross-build support) and only merge the other commits?